module.d.ts

```diff
@@ -434,7 +435,7 @@ declare module '@solana/web3.js' {
     authorizedPublicKey: PublicKey | null;
 
     constructor(
-      opts?: TransactionInstructionCtorFields,
+      opts: TransactionInstructionCtorFields,
```

Comment:
This is more restrictive than the code implementation but this API will be removed soon.